widgets/matplotlib/measurement/energy_spectrum.py

```diff
@@ -53,11 +53,17 @@
 from widgets.matplotlib.base import MatplotlibWidget
 
 
+MEASURED_SUM_SPECTRUM_LINE_STYLE = "dashed"
+SIMULATED_SUM_SPECTRUM_LINE_STYLE = "dotted"
+SUM_SPECTRUM_LINE_WIDTH = 2
+Z_ORDER = 100
```

Comment:
These should be together with `DEFAULT_LINE_STYLE` (either in `MatplotlibEnergySpectrumWidget` or here).

Reply:
Check